### PR TITLE
Schema updates, drop some fields

### PR DESCRIPTION
**Pull request by matthewhanson**
- Define all fields to be Item properties (which can then be defined per asset as needed)
- removes date_stored, which seems redundant with created/updated for assets (#6)
- removes manager, seems redundant with providers array (#5)
- updated JSON schema to follow same structure as in stac-extensions template
- add contributing chapter
- makes requester_pays field not required, defaults to False

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,40 +5,37 @@
...
 
 ## Item Fields
+
 | Field Name  | Type   | Description |
 | ----------- | ------ | ----------- |
 | storage:min_tier_duration   | integer  | number of days for the shortest time tier restriction on access of an asset. |
 | storage:max_tier_duration   | integer  | number of days for the longest time tier restrictions on access of an asset. |
 | storage:archived            | bool    | descriptor for whether the data is "properly" archived according to implementers discretion |
-
-## Asset Fields
```
Comment (davidraleigh) [inline README.md:27]: Does this mean everything is under Item now?

File: README.md [hunk 2]
```diff
@@ -5,40 +5,37 @@
...
-
-| Field Name  | Type   | Description |
-| ----------- | ------ | ----------- |
 | storage:platform              | string    | (REQUIRED) The [cloud provider](#providers) where data is stored |
```
Comment (davidraleigh) [inline README.md:27]: `platform` at Item level won't work if you have multiple assets in different cloud platforms

File: README.md [hunk 3]
```diff
@@ -5,40 +5,37 @@
...
 | storage:region                | string    | (REQUIRED) The region where the data is stored. Relevant to speed of access and inter region egress costs (as defined by PaaS provider) |
-| storage:bucket                | string    | The bucket for the asset, used along with object path |
+| storage:bucket                | string    | The bucket for the asset(s), used along with object path |
 | storage:object_path           | string    | The object_path for the asset, used along with bucket |
-| storage:requester_pays        | bool      | (REQUIRED) Is the data requester pays or is it data manager/cloud provider pays |
+| storage:requester_pays        | bool      | Is the data requester pays or is it data manager/cloud provider pays. *Defaults to false* |
 | storage:tier                  | string    | The title for the tier type (as defined by PaaS provider) |
 | storage:tier_duration         | integer   | Minimum storage duration (in days) required before additional fees |
-| storage:date_stored           | string    | Date and time the corresponding asset placed into the current storage tier (relevant for tier_duration > 0). Format is RFC 3339. |
```
Comment (davidraleigh) [inline README.md:39]: This is useful for keeping track of budgeting and how often a data set is pulled out of a "coldish" storage before it's "ready". but if you want to drop it, maybe it's too niche.

File: README.md (hunk 1 above)
Comment (matthewhanson) [reply README.md:27]: @davidraleigh Assets can now specify any STAC property that can be at the Item level, see https://github.com/radiantearth/stac-spec/blob/master/item-spec/item-spec.md#additional-fields-for-assets
So this allows any asset to use these properties as well. By having them at the Item level you can specify a single default value, but can override in any of the assets.

File: README.md (hunk 2 above)
Comment (matthewhanson) [reply README.md:27]: As per above comment, you can specify `storage:platform` at the Item level and/or the asset level

File: README.md (hunk 1 above)
Comment (davidraleigh) [reply README.md:27]: I just saw this: ["While these are all valid properties on an Item, they will typically be defined per-asset. If a field applies equally"](https://github.com/stac-extensions/storage/pull/10/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R36)  

I didn't know that was possible. cool

File: README.md [hunk 4]
```diff
@@ -5,40 +5,37 @@
...
 | storage:first_byte_latency    | string    | approximate time unit (milliseconds, minutes or hours) for accessing first byte of data |
 
+While these are all valid properties on an Item, they will typically be defined per-asset. If a field applies equally
+to all assets (e.g., storage:platform=AWS if all assets are on AWS), then it should be specified in Item properties.
```
Comment (davidraleigh) [inline README.md:37]: cool

File: README.md (hunk 3 above)
Comment (matthewhanson) [reply README.md:39]: It seems pretty specific, but also incomplete if you really wanted to do through tracking of data management. Seems like that should be tracked with an internal extension or as part of the system that manages storage locations.

**Pull request merged**